src/stores/infra/postgres/postgres.go

```diff
@@ -0,0 +1,6 @@
+package postgres
+
+//go:generate mockgen -source=postgres.go -destination=mocks/postgres.go -package=mocks
```

Comment:
WHat is the propose of mocking this empty interface?

Reply:
the interface is not going to stay empty... This is just a skeleton